src/action-bar.scss

```diff
@@ -43,25 +41,47 @@ $block: #{$fd-namespace}-action-bar;
     }
   }
 
+  @include fd-screen(s) {
+    padding-left: map_get($fd-action-bar-padding-x, "s");
+    padding-right: map_get($fd-action-bar-padding-x, "s");
+  }
+
+  @include fd-screen(m) {
+    padding-left: map_get($fd-action-bar-padding-x, "s");
+    padding-right: map_get($fd-action-bar-padding-x, "s");
+  }
+
+  @include fd-screen(l) {
+    padding-left: map_get($fd-action-bar-padding-x, "l");
+    padding-right: map_get($fd-action-bar-padding-x, "l");
+  }
+
+  @include fd-screen(xl) {
+    padding-left: map_get($fd-action-bar-padding-x, "xl");
+    padding-right: map_get($fd-action-bar-padding-x, "xl");
+  }
+
   // BLOCK BASE *******************************************
   @include fd-reset();
 
   background-color: $fd-action-bar-background-color;
   min-height: $fd-action-bar-height;
-  padding-right: 0;
-  padding-left: 0;
+  padding-top: 0.5rem;
+  padding-bottom: 0.5rem;
   display: flex;
+  flex-direction: row;
   align-items: baseline;
+  word-wrap: break-word;
+  justify-content: space-between;
+  padding-left: map_get($fd-action-bar-padding-x, "m");
+  padding-right: map_get($fd-action-bar-padding-x, "m");
 
   &__header {
+    flex-grow: 1;
+    width: 75%;
```

Comment:
this code should be after the reset function

Reply:
Placed after reset